views/ViewChart.react.js

```diff
@@ -7,7 +7,7 @@ import { Typography } from '@material-ui/core'
 import { chartStyles } from './styles/chart_styles'
 import AppLayout from './layouts/AppLayout'
 import BarGraph from './components/Graphs/BarGraph'
-import GraphTable from './components/Graphs/GraphTable'
+import GraphTable from './components/GraphTable/GraphTable'
```

Comment:
the component should live in an `index.jsx` file so we don't need to repeat the name here.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/193/commits/325e6635a9f09c0b3ce28c6d2a5ab8bfa1369601